packages/generators/src/init-generator.ts

```diff
@@ -70,6 +73,24 @@ export default class InitGenerator extends CustomGenerator {
         }
 
         await handlers[this.template].questions(this, Question);
+
+        // Handle installation of prettier
+        try {
+            // eslint-disable-next-line node/no-extraneous-require
+            require.resolve('prettier');
+        } catch (err) {
+            const { installPrettier } = await Question.Confirm(
+                this,
+                'installPrettier',
+                'Do you like to install prettier to format generated configuration?',
+                true,
+                false,
+            );
+
+            if (installPrettier) {
+                this.dependencies.push('prettier');
```

Comment:
should be devDeps

Reply:
Every dep in this.dependencies is installed as devDep.